import org.apache.hadoop.fs.PathIOException;
public class FunctionUse7064 {
public void funcUse() {
PathIOException pathioexception = new PathIOException();
pathioexception.getPath();
}
}